docs/modules/ROOT/pages/static-site-generators.adoc

```diff
@@ -0,0 +1,94 @@
+= Static Site Generators
+
+Asciidoctor fundamentally converts one source document into one output document.
+To convert more than one document, you need some amount of infrastructure.
+Simple situations may find it convenient to use built-in Asciidoctor capabilities or scripts, but are unlikely to be appropriate for any but the smallest sites.
+Static site generators organize the conversion of multiple source documents into a functioning static web site.
+
+== What does a Static Site Generator (ssg) do?
+
+A static site generator (ssg) gathers sources from one or more locations, transforms them, and writes the results into a file system, concrete or virtual, ready for deployment to a web server.
+Typical source locations are the local file system, remote Git repositories, and artifact repositories such as those used by Maven, e.g. Nexus.
+The most common transformations are converting source from text representations such as AsciiDoc or Markdown to HTML and embedding the results in additional HTML markup to supply a site theme and navigation.
+Other transformations include generating diagram images from text sources and optimizing images.
+Mavigation resources can be generated from explicit source or automatically.
+
+Some static site generators include features specifically targeted to particular website styles such as blogs.
+
+== What is Asciidoctor's role in static site generation?
+
+Asciidoctor is used in static site generators to transform source documents in Asciidoc into html.
+Most usually this is simply embedded in "outer html" supplied by a template, but it can also be further transformed, as done by nav file processing in Antora.
```

Comment:
```suggestion
Usually the content is embedded in "outer html" supplied by a template, but it can also be further transformed, as done by nav file processing in Antora.
```

Reply:
I don't think we should mention Antora at this point. If you are not familiar with SSG you are probably not familiar with Antora so this does not help.

Reply:
I agree, though I think it would be fair to say somewhere that AsciiDoc is usually use for the main page content, but can be used in other, novel ways by the SSG.

Reply:
Yes that's a good idea.
What you suggested could replace this sentence:

"AsciiDoc is usually use for the main page content, but can be used in other, novel ways by the SSG."

We could even mention that some SSGs are using AsciiDoc to define navigation between pages.

Reply:
How about...
"Asciidoctor is used in static site generators to transform source documents in AsciiDoc into HTML.
Usually this is embedded in "outer HTML" supplied by a template to produce an output page.
Some static site generators also use specific AsciiDoc input for novel purposes, such as to generate navigation resources."